test/sqllogictest/persist-fast-path.slt

```diff
@@ -195,15 +195,15 @@ EOF
 # an arbitrary number of records to find one that matches.
 
 query T multiline
-EXPLAIN OPTIMIZED PLAN WITH (humanized expressions) AS VERBOSE TEXT FOR SELECT * from numbers WHERE value > mz_now() LIMIT 10;
+EXPLAIN OPTIMIZED PLAN WITH (humanized expressions) AS VERBOSE TEXT FOR SELECT * from numbers WHERE value > mz_now()::text::int8 / 100000000000000 LIMIT 10;
```

Comment:
Took me a second to notice the case here inducing the type change below. Looks good.